.github/workflows/cd.yml

```diff
@@ -185,7 +185,29 @@ on:
         type: string
         required: false
 
-      # Artifacts attestation for build provenance
+      # Feature toggle: plugin-validator
+      run-plugin-validator:
+        description: Whether to run plugin-validator.
+        type: boolean
+        required: false
+        default: false
+      plugin-validator-config-file:
```

Comment:
this name is rather confusing, `plugin-validator-config-path` and `plugin-validator-config-file` can be confused as both accepting a path

what if we call them:

`plugin-validator-config` and the other one `plugin-validator-config-path`